packages/astro/test/test-utils.js

```diff
@@ -139,9 +139,13 @@ export async function loadFixture(inlineConfig) {
 	let devServer;
 
 	return {
-		build: (opts = {}) => build(settings, { logging, telemetry, ...opts }),
+		build: (opts = {}) => {
+			process.env.NODE_ENV = 'production';
```

Comment:
Vite 4 doesn't mutate `NODE_ENV` anymore, unless it's `undefined`. So if we run the dev server, and then build, build would run in `development` `NODE_ENV` instead, which broke the MDX test.

This shouldn't affect users in practice, but internally for our tests we need to make sure `NODE_ENV` is the right value we're testing.